src/index.js

```diff
@@ -159,20 +159,10 @@ class App extends Component {
                 <Route path="/complete" component={Complete} />
                 <Route path="/error" component={CompleteError} />
                 <Route path="/cra" component={Cra} />
-                <SentryRoute
-                  path="/employee/:slug"
-                  component={Employee}
-                ></SentryRoute>
-                <SentryRoute
-                  path="/product/:id"
-                  component={Product}
-                ></SentryRoute>
-                <Route path="/products">
-                  <Products />
-                </Route>
-                <Route path="/products-join">
-                  <ProductsJoin />
-                </Route>
+                <SentryRoute path="/employee/:slug" component={Employee}></SentryRoute>
+                <SentryRoute path="/product/:id" component={Product}></SentryRoute>
+                <Route path="/products" component={Products} />
```

Comment:
passing the Products component into the Route's tag like this is 100% required, or else `this.props.history` is always null.

Can't do:
```
 <Route path="/products">
                  <Products />
                </Route>
```
Must do:
```
<Route path="/products" component={Products} />
```